Module/tide_utils2.py

```diff
@@ -0,0 +1,193 @@
+        PortTownsend = PortTownsend.append(PortTownsend_2016)
+        PortTownsend.index.rename('datetime', inplace=True)
+        return PortTownsend
+
+
+def load_Port_Angeles(datadir):
+    """
+    Function to load the Port Angeles tidal station data from 2015
+    & 2016. Supply the directory to where the csv files with the
+    data are saved. Returns None if files are not located in
+    specified directory.
+    """
+    if not glob.glob(os.path.join(datadir, '*PortAngeles.csv')):
+        return None
+    else:
+        # Load the Port Angeles tidal data and put into one dataframe
+        PortAngeles_2014 = pd.read_csv(datadir +
+                                       '2014_PortAngeles.csv',
+                                       parse_dates=['Date Time'],
+                                       index_col=['Date Time'])
+        PortAngeles_2015 = pd.read_csv(datadir +
+                                       '2015_PortAngeles.csv',
+                                       parse_dates=['Date Time'],
+                                       index_col=['Date Time'])
+        PortAngeles_2016 = pd.read_csv(datadir +
+                                       '2016_PortAngeles.csv',
+                                       parse_dates=['Date Time'],
+                                       index_col=['Date Time'])
+        PortAngeles = PortAngeles_2014.append(PortAngeles_2015)
+        PortAngeles = PortAngeles.append(PortAngeles_2016)
+        PortAngeles.index.rename('datetime', inplace=True)
+        return PortAngeles
+
+
+def load_tide_data(datadir):
+    """
+    Upper level load function for the Tide Data.
+    Datadir is the directory where the data .csv files are saved
+    """
+    NeahBay = load_Neah_Bay(datadir)
+    PortAngeles = load_Port_Angeles(datadir)
+    PortTownsend = load_Port_Townsend(datadir)
+    if NeahBay is None:
+        return None
+    elif PortAngeles is None:
+        return None
+    elif PortTownsend is None:
+        return None
+    else:
+        return NeahBay, PortAngeles, PortTownsend
+
+
+def create_tide_dataset(NeahBay, PortAngeles, PortTownsend):
+    """
+    Function takes in the tidal station dataframes and returns
+    an Xarray Dataset with the tidal station data
+    """
+    NB = xr.DataArray(NeahBay['Water Level'], dims='datetime')
+    PA = xr.DataArray(PortAngeles['Water Level'], dims='datetime')
+    PT = xr.DataArray(PortTownsend['Water Level'], dims='datetime')
+    Tides = xr.Dataset({'NeahBay': NB, 'PortAngeles': PA,
+                       'PortTownsend': PT})
+    return Tides
+
+
+def plot_tide_data(Tides,time1,time2):
+    """
+    Function that allows to pass through variables to
+    the interactive widget.
+    Inputs:
+        Tides - Xarray DataSet with the tidal station data
+        time1 - start time to slice the tidal data
+        time2 - end time to slice the tidal data
+    """
+    from ipywidgets import interact
+    import ipywidgets as widgets
+
```

Comment:
I assume these are here because widgets doesn't work if you load at the file head? Otherwise, I'd say move these up top